addons/medical_treatment/XEH_postInit.sqf

```diff
@@ -21,3 +21,60 @@ if (isServer) then {
 [QGVAR(actionCheckPulseLocal), FUNC(actionCheckPulseLocal)] call CBA_fnc_addEventHandler;
 [QGVAR(actionCheckBloodPressureLocal), FUNC(actionCheckBloodPressureLocal)] call CBA_fnc_addEventHandler;
 [QGVAR(actionPlaceInBodyBag), FUNC(actionPlaceInBodyBag)] call CBA_fnc_addEventHandler;
+
+// Notification lets the player known if they're being treated
+[QGVAR(initiated), {
+    params ["_caller", "_target"];
+
+    // Notifications don't make sense for self-healing or when dead
+    if (_target != ACE_player || {_target == _caller} || {!alive _target}) exitWith {};
+
+    // Can't tell who is providing aid when you're unconscious
+    private _name = if IS_UNCONSCIOUS(_target) then {
+        LLSTRING(UnknownPerson);
+    } else {
+        name _caller
+    };
+
+    QGVAR(notification) cutText [format [LLSTRING(Notification), _name], "PLAIN DOWN"];
```

Comment:
I'm not to fond of the `cutText` bit. Do we have alternatives for displaying the notification?

Maybe the `displayIcon` framework might work, as an optional setting? This is a bit less in your face, which might be nice.

Reply:
I'm not familiar with the `displayIcon` stuff and will need to take a look, I do like the idea of allowing icons vs text.

As for an alternative to `cutText` the only other options I can think of are to add a custom `cutRsc` that's more subtle or using a hint/displayStructuredText (which I'd be against as they're even more in your face).